ProjectBank/Client/Pages/MyProjects.razor

```diff
@@ -0,0 +1,21 @@
+@page "/user"
+
+<PageTitle>My Projects</PageTitle>
+
+<h1>My Projects</h1>
+
+<ul>
+    @foreach (var project in user.Projects)
+    {
+        <li>@project.Title</li>
+    }
+</ul>
+
+@code {
+
+    User user = new User("Alice", "Alice@itu.dk", "I<3Algo")
```

Comment:
```suggestion
    User user = new User("Alice", "Alice@email.com", "password123")
```
Something like this might make it easier for other people to understand that this parameter is for a password